src/lib/assets/assets.test.ts

```diff
@@ -165,23 +164,45 @@ jest.mock("./index", () => ({
 }));
 
 describe("Assets", () => {
-  const moduleConfig: ModuleConfig = {
-    uuid: "test-uuid",
-    alwaysIgnore: {
-      namespaces: ["zarf"],
-    },
-    peprVersion: "0.0.1",
-    appVersion: "0.0.1",
-    description: "A test module",
-    webhookTimeout: 10,
-    onError: "reject",
-    logLevel: "info",
-    env: {},
-    rbac: [],
-    rbacMode: "scoped",
-    customLabels: {},
-  };
-  const assets = new Assets(moduleConfig, "/tmp", ["secret1", "secret2"], "localhost");
+  let moduleConfig: ModuleConfig;
+  let assets: Assets;
+  let consoleErrorSpy: ReturnType<typeof jest.spyOn>;
+
+  const createMockWebhookGenerator = () =>
+    jest
+      .fn<() => Promise<V1MutatingWebhookConfiguration | V1ValidatingWebhookConfiguration | null>>()
+      .mockResolvedValue(new kind.MutatingWebhookConfiguration());
+
+  const createMockWatcher = () =>
+    jest.fn<() => kind.Deployment | null>().mockReturnValue(new kind.Deployment());
+
+  const createMockModuleSecret = () =>
+    jest.fn<() => kind.Secret>().mockReturnValue(new kind.Secret());
```

Comment:
My declaring generator functions here, we avoid declaring the same mock over and over.